doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -204,33 +228,36 @@ a. Download and extract the Service OS kernel
       [    0.000000] Hypervisor detected: ACRN
       [    0.862942] ACRN HVLog: acrn_hvlog_init
 
+
 Prepare the User OS (UOS)
 *************************
 
 We are using a User OS based on `Clear Linux`_.
+supposing to use same release with Service OS - 24040
```

Comment:
Change these two sentences to be:

    For the User OS, we are using the same `Clear Linux`_ release version as the Service OS.

Reply:
okay